ml-agents/mlagents/trainers/action_info.py

```diff
@@ -0,0 +1,9 @@
+from typing import NamedTuple, Any, Dict, Optional
+
+
+class ActionInfo(NamedTuple):
+    action: Any
+    memory: Any
+    text: Any
```

Comment:
@vincentpierre @awjuliani `take_action` for both `BCTrainer` and `PPOTrainer` returns None for this entry of the tuple. Is there a point in even having this as a return from `take_action`?

Reply:
Per our discussion, we'll deprecate this later.